src/the_program_tunserver.cpp

```diff
@@ -206,6 +206,10 @@ std::tuple<bool,int> c_the_program_tunserver::base_options_commands_run() {
 }
 
 int c_the_program_tunserver::main_execution() {
+	_mark("Main execution of the old-loop");
+	_warn("Remember, that this old-loop code is NOT secured as new-loop code, e.g. is not droping CAP/root privileges!");
```

Comment:
It's about old-loop not being secured in general, not only about the Caps.

:no_entry: